icons/split-rows.svg

```diff
@@ -0,0 +1,18 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M12 21V3" />
+  <path d="M16.5 15V9" />
```

Comment:
These two lines are off-grid, which makes them appear blurry on low resolution screens:
<img width="316" alt="image" src="https://github.com/lucide-icons/lucide/assets/17746067/e9620004-1a59-4bf2-97b8-0c3b214e28fc">

All in all, this icon does feel a bit too dense to begin with, maybe we could go with a simple inverse of the merge variant instead?
![image](https://github.com/lucide-icons/lucide/assets/17746067/d58c61ba-c598-4a8b-b216-bc292892add3)

Some other ideas include these, but I'm not super thrilled about any of these.
![image](https://github.com/lucide-icons/lucide/assets/17746067/fcc1c8f1-d96e-476d-abcc-f6a9e36a3801)

Reply:
Also, `split-square-horizontal` and `split-square-vertical` could be possible candidates for this use case.

Reply:
I like the first icon. Is it in the `lab`?

Reply:
The first one looks cluttered. 

![split-rows](https://github.com/lucide-icons/lucide/assets/137697011/55e40542-cf17-4d1a-bf81-278fdeec2ec4)